iib/workers/tasks/build.py

```diff
@@ -378,12 +428,16 @@ def _prepare_request_for_build(
         currently built for; if ``from_index`` is ``None``, then this is used as the list of arches
         to build the index image for
     :param list bundles: the list of bundles to create the bundle mapping on the request
-    :return: a dictionary with the keys: arches, binary_image_resolved, and fom_index_resolved.
+    :return: a dictionary with the keys: arches, binary_image_resolved, bundles_resolved,
+        and from_index_resolved.
     :raises IIBError: if the container image resolution fails or the architectures couldn't be
         detected.
     """
     if bundles is None:
         bundles = []
```

Comment:
You should use `resolved_bundles` instead of `bundles` when getting the bundle mapping below.